### PR TITLE
Feature/storage/blob tags

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Review by kasobol-msft — APPROVED** [review]

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json [hunk 1]
```diff
@@ -8568,6 +8610,160 @@
...
         }
       ]
+    },
+    "/{containerName}/{blob}?comp=tags": {
+      "get": {
+        "tags": [
+          "blob"
+        ],
+        "operationId": "Blob_GetTags",
+        "description": "The Get Tags operation enables users to get the tags assocaited with a blob.",
```
Comment (gapra-msft) [inline specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json:8620]: typo: associated

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json [hunk 2]
```diff
@@ -8568,6 +8610,160 @@
...
+          {
+            "$ref": "#/parameters/ClientRequestId"
+          },
+          {
+            "$ref": "#/parameters/BlobTagsBody"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The blob tags with applied",
```
Comment (gapra-msft) [inline specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json:8718]: The blob tags with applied what?

**Review by gapra-msft — APPROVED** [review]

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json (hunk 1 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json:8620]: fixed.

File: specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json (hunk 2 above)
Comment (seanmcc-msft) [reply specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json:8718]: fixed.

**Pull request merged**